packages/app/src/app/components/Alert/index.tsx

```diff
@@ -1,23 +1,35 @@
 import React from 'react';
-
 import { Button } from '@codesandbox/common/lib/components/Button';
 import { Container, Title, Text, Buttons } from './elements';
 
-function Alert({ title, body, onCancel, onDelete, ...props }) {
-  return (
-    <Container {...props}>
-      <Title>{title}</Title>
-      <Text>{body}</Text>
-      <Buttons>
-        <Button small block secondary onClick={onCancel}>
-          Cancel
-        </Button>
-        <Button small block danger onClick={onDelete}>
-          Delete
-        </Button>
-      </Buttons>
-    </Container>
-  );
+interface IAlertProps {
```

Comment:
We're going to stick with `interface` over `type` because interfaces are what are used in the TypeScript language docs for JSX, in Microsoft's React TypeScript starter, as well as various other tutorials. There doesn't appear to be a big advantage to one over the other, but if that's what the language docs use, better to stay consistent.

Reply:
I really think we should use `type` for the reasons explained in [this Medium](https://medium.com/@martin_hotell/interface-vs-type-alias-in-typescript-2-7-2a8f1777af4c) post tho.

It basically comes down to: "Use `type` for internal things, use `interface` if you're exporting them"

Reply:
I read that too and I don't find that to be a compelling reason to use `type` over `interface`. At the time that was written in 2018 I believe his suggestion of using HOCs for type extensions was a good point, but now with hooks I believe we'll be using less HOCs overall. So when comparing the arguments of public versus private types against having code be consistent with official documentation examples, I think prefer `interface` wins out here as this is an open source project and we want it to be as approachable as possible for contributors. As stated by the author of the post, use whatever suits your team best.

Reply:
The "use what the official docs do" argument doesn't really fit imho, since the article makes it clear that the official docs about `type` vs. `interface` (see [here](http://www.typescriptlang.org/docs/handbook/advanced-types.html#interfaces-vs-type-aliases)) is outdated too

@CompuIves @SaraVieira What're your opinions about this one?